tool/bazelinstall/rbe.sh

```diff
@@ -2,18 +2,20 @@
 
 set -e
 
-if [[ -n "$BAZEL_CACHE_URL" ]]; then
+if [[ -n "$BAZEL_CACHE_CREDENTIAL" ]] && [[ -n "$BAZEL_CACHE_URL" ]]; then
     echo "Installing remote cache credential..."
-    BAZEL_CACHE_CREDENTIAL=/opt/credentials/
-    echo "A remote cache credential is found and will be saved to $BAZEL_CACHE_CREDENTIAL. Artifact and test results will be cached remotely."
-    sudo mkdir -p $BAZEL_CACHE_CREDENTIAL && sudo chmod a+rwx $BAZEL_CACHE_CREDENTIAL
-    ( echo "build --remote_cache=$BAZEL_CACHE_URL"; echo "test --remote_cache=$BAZEL_CACHE_URL"; echo "run --remote_cache=$BAZEL_CACHE_URL" ) >> "$BAZEL_CACHE_CREDENTIAL/bazel-remote-cache.rc"
-    echo "The RBE credential has been installed!"
-    echo "Configuring Python..."
-    pyenv global system system
+    BAZEL_CACHE_CREDENTIAL_LOCATION=./credential.json
+    echo "A remote cache credential is found and will be saved to $BAZEL_CACHE_CREDENTIAL_LOCATION. Artifact and test results will be cached remotely."
+    BAZEL_CACHE_CREDENTIAL_JSON=$(echo "$BAZEL_CACHE_CREDENTIAL" | base64 -d $BAZEL_CACHE_CREDENTIAL_LOCATION)
+    if [[ -n "$BAZEL_CACHE_CREDENTIAL_JSON" ]]; then
+      ( echo "build --remote_cache=$BAZEL_CACHE_URL --google_credentials=$BAZEL_CACHE_CREDENTIAL_LOCATION"; echo "test --cache_test_results=no" ) >> ./.bazel-remote-cache.rc
```

Comment:
We don't want to cache test results by default as a result of this rc file having its build instructions inform the test instructions. See the option precedence instruction at: https://bazel.build/run/bazelrc